app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  validates :name, presence: true
+  has_many :events, inverse_of: :creator
+  has_many :enrollments
+  has_many :attended_events, through: :enrollments, source: :event
+
+  scope :past, -> { where('date < ?', Date.today) }
+  scope :upcoming, -> { where('date >= ?', Date.today) }
```

Comment:
We don't need this for the user model. We don't have a date filed for a user. We only needed this for the event model.

Reply:
Hi.  this requirement says:

- [ ] [REQUIRED] Navigation links are present at the top of the page  (These links should be displayed once the user is logged in: the event's index page, the event's show page, The user's show page).

We include the screenshot, you can check the navbar with the links, is that what you are saying? sorry but that is what we understand.
 
![image](https://user-images.githubusercontent.com/29665697/104811885-90d8bc00-57cc-11eb-9b40-7e40277fa2d4.png)

Reply:
Hello team!!  Thanks for your reply. However, if you read my comment carefully, I mentioned that: event's index page, event's show page and user's show page should be also in the navigation bar.